public/app/features/explore/Logs/LogsContainer.tsx

```diff
@@ -206,6 +207,7 @@ class LogsContainer extends PureComponent<LogsContainerProps> {
             addResultsToCache={() => addResultsToCache(exploreId)}
             clearCache={() => clearCache(exploreId)}
             eventBus={this.props.eventBus}
+            isFilterLabelActive={this.props.isFilterLabelActive}
```

Comment:
Do you know if Table component supports something similar? I think it also supports filtering.

Reply:
Not that I'm aware of, I've only seen sort in table views. E.g.

![imagen](https://github.com/grafana/grafana/assets/1069378/ba281af3-dcd7-46bd-b335-b664795c6316)

Reply:
Prometheus supports it. It's a bit hidden coz it's shows up on however over a row, but would be nice to have it consistent:

<img width="1520" alt="Screenshot 2023-06-23 at 09 12 59" src="https://github.com/grafana/grafana/assets/745532/6a6cb1cd-f0c6-4166-a231-112706ba01ef">

Reply:
Oh interesting! What would you suggest here? To also pass the new prop to this table component, or to consider adding support of the whole thing for Prom?

Reply:
Good question :D I think it would be great to add support for it in Table component. I guess it's something out of the scope for this PR, so maybe it's worth creating an issue to discuss requirements and ownership?

Reply:
Sounds good! https://github.com/grafana/grafana/issues/70584